lib/ui/text.dart

```diff
@@ -432,14 +432,43 @@ enum TextDecorationStyle {
   wavy
 }
 
+/// {@macro dart.ui.leadingDistribution}
+enum LeadingDistribution {
+  /// Distributes the ["leading"](https://www.w3.org/TR/CSS2/visudet.html#leading)
+  /// space of the text proportionally above and below the text, to the
+  /// font's ascent/discent ratio.
+  ///
+  /// {@template dart.ui.leading}
+  /// The leading space of a text run is defined as
```

Comment:
"leading" is a noun, not an adjective. (It's pronounced like the metal Pb, as opposed to like the opposite of "following")

Reply:
(which is to say, it's "the leading", not "the leading space".)